src/app/layout/navbar/navbar.component.scss

```diff
@@ -24,10 +24,14 @@ nav {
 
 #sw-logotype {
   object-fit: contain;
-  height: 50px;
+  height: 60px;
   padding-top: 3px;
 
+  @media #{$small-and-down} {
+    height: 50px;
+  }
+
   @media #{$medium-and-up} {
-    padding: 7px 0px 0px 30%;
+    padding: 7px 0px 0px 7%;
   }
 }
```

Comment:
mobile first: the default height should be 50px, and only resize in medium and up... Also, the padding top in medium and up was set to 7px because the height of the navbar is 64px, if the image height is 50px, that leaves 14px to be distributed up and down, so the half of it should go to the top, which is 7px... If you change the height of the logo, this rule breaks

Reply:
When using 60px for the logo height, padding top should be 2px